### DISCUSSION TITLE
Cannot complete nk.epochs_create

### DISCUSSION THREAD

**Discussion by RyotaTKN** [neuropsychology/NeuroKit#655] [Q&A] [answered]
Hello,

I am trying to analyze my ECG data. There are 100 events (trials). I'd like to investigate the HRV in each event.
When I run the following code, the zsh is killed just like the pic below. How can I deal with this problem?
```
epochs = nk.epochs_create(df, events)
```

If necessary, I will send you the data and code.

<img width="1198" alt="image" src="https://user-images.githubusercontent.com/58549690/173224698-3869937d-245b-46d6-b989-57f8017431f6.png">

**Comment by DominiqueMakowski** [comment, marked as answer]
It seems like you forgot to import neurokit using `import neurokit2 as nk`

**Reply by RyotaTKN** [reply]
Thank you for your reply! 

This error occurred because the terminal is down. (I had to start from the first "import" because it was initialized)

I solved the problem by myself. I had too much data, so I did a down sampling (sampling_rate = 100) and it ran correctly!
